src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -90,16 +57,57 @@ public class BatchHandler : DatabaseRequestHandler
 
                 HttpContext.Response.StatusCode = (int)HttpStatusCode.Created;
 
-                using (var writer = new BlittableJsonTextWriter(ctx, ResponseBodyStream()))
+                using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
                 {
-                    ctx.Write(writer, new DynamicJsonValue
+                    context.Write(writer, new DynamicJsonValue
                     {
                         ["Results"] = mergedCmd.Reply
                     });
                 }
             }
         }
 
+        private async Task<MergedBatchCommand> CreateBatchCommand(DocumentsOperationContext context)
+        {
+            var command = new MergedBatchCommand {Database = Database};
+
+            if (HttpContext.Request.ContentType != null &&
+                HttpContext.Request.ContentType.StartsWith("multipart/mixed", StringComparison.OrdinalIgnoreCase))
```

Comment:
Can it actually be anything _but_ "multipart/mixed" exactly?
The reason I'm asking is that I think that the browser will send it with `text/json` or some such, no?